### ISSUE TITLE
[NodeJS][HTML] [Duplicate HTML ids]

### ISSUE THREAD

**Issue by scottlovegrove** [microsoft/AdaptiveCards#5675]
# Platform

What platform is your issue or question related to? (Delete other platforms).

- [ ] .NET HTML
- [ ] .NET WPF
- [ ] Android
- [ ] iOS
- [x] JavaScript
- [ ] UWP

# Author or host

Host

If you're an author, who are you sending cards to?

# Version of SDK

2.7.0

# Details

We've been looking at a way to be able to allow focus on a particular input element when a card has been loaded. We were going to do this by providing an id and then using `document.getElementById()` to get the item and call `focus()` on it, but we've run into an issue where the renderer is applying the provided id to the input item, but _also_ to the container div around it. 

Because the HTML guidance is all ids should be unique, calling `document.getElementById()` will only ever get the first item it finds, which in the case of the adaptive card is the containing div, and _not_ the input. 

Is this a bug? If not, what was the reasoning behind breaking HTML guidance for this?

**Comment by jonmill** [comment]
Thanks for bringing this up! This definitely sounds like a bug; we'll dig in and take a look

**Comment by jankratochvilcz** [comment]
@jonmill is there any update you could share?

**Comment by jonmill** [comment]
@jankratochvilcz - We're trying to get to it in May but we aren't 100% positive that it will fit yet. If you're willing or able to, we'd welcome you contributing the fix! :)

**Comment by scottlovegrove** [comment]
@jonmill I took a look at this over the weekend, and I can confirm this issue can be closed. The error was on my part. I'm doing some custom rendering for TextInput and using a one of our own components, and whilst doing that, I was assigning the id to that component, which is what resulted in the duplicate ids appearing. When I commented out the registry call and used the native sdk rendering for input, the issue wasn't there.

Apologies for this.
